src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const getChatLog = (props) => {
+    return props.entries.map((chatEntry) => {
+      return (
+        <ChatEntry
+          key={chatEntry.id}
+          id={chatEntry.id}
+          sender={chatEntry.sender}
+          body={chatEntry.body}
+          timeStamp={chatEntry.timeStamp}
+          liked={chatEntry.liked}
+          onLike={props.onLike}
+          remoteSend={props.remoteSender===chatEntry.sender}
```

Comment:
This is a very nice way to encapsulate data away from the lower levels of your hierarchy. A `ChatEntry` doesn't _need_ to know which sender is the remote one, so just a boolean telling it if it is the remote sender is fine.